rom/dev/src/rom.ld

```diff
@@ -19,7 +19,8 @@ ENTRY(_start)
 ROM_ORG          = 0x00000000;
 ICCM_ORG         = 0x40000000;
 DCCM_ORG         = 0x50000000;
-DATA_ORG         = 0x50004000;
+/* Avoid overlap with other long-lived DCCM structures */
```

Comment:
Please update memory_layout.rs in common\src with this change.

Reply:
Actually I don't think this change is needed anymore. I don't see any overlap. I'll remove this.